tox.ini

```diff
@@ -8,14 +8,16 @@ deps =
     jsonschema
     pytest
     pyyaml
+    pytest-cov
 whitelist_externals =
     make
 setenv =
 	CACHITO_TESTING=true
 commands =
     py.test -vvv \
         --ignore tests/integration \
-        {posargs}
+        --cov-config .coveragerc --cov=./ --cov-report term \
```

Comment:
Try this and see if it works.
```suggestion
        --cov-config .coveragerc --cov=cachito --cov-report term \
```

Reply:
@mprahl Thank you! I tried command `tox` with `--cov=cachito` parameter even before. But I got 0% for all multiline-files. I am wondering where are diffs with [iib project](https://github.com/release-engineering/iib/blob/master/tox.ini)..?

Reply:
Elina, tox uses the installed files to run your tests, while we are pointing pytest-cov to the current working directory.

The solution here would be to set cov to `--cov={envsitepackagesdir}/cachito`

Reply:
@athos-ribeiro Wow, thank you!

Reply:
@athos-ribeiro No, it doesn't work. It checks only `./tox` files: 


```----------- coverage: platform linux, python 3.7.7-final-0 -----------
Name                                                                                                              Stmts   Miss  Cover
-------------------------------------------------------------------------------------------------------------------------------------
.tox/py37/lib/python3.7/site-packages/cachito/__init__.py                                                             0      0   100%
.tox/py37/lib/python3.7/site-packages/cachito/errors.py                                                               3      0   100%
.tox/py37/lib/python3.7/site-packages/cachito/paths.py                                                               29      0   100%
.tox/py37/lib/python3.7/site-packages/cachito/web/__init__.py                                                         2      0   100%
.tox/py37/lib/python3.7/site-packages/cachito/web/api_v1.py                                                         222      7    97%
.tox/py37/lib/python3.7/site-packages/cachito/web/app.py                                                             61     15    75%
.tox/py37/lib/python3.7/site-packages/cachito/web/auth.py                                                            24      1    96%
.tox/py37/lib/python3.7/site-packages/cachito/web/config.py                                                          41      0   100%
.tox/py37/lib/python3.7/site-packages/cachito/web/content_manifest.py                                                52      7    87%
.tox/py37/lib/python3.7/site-packages/cachito/web/docs.py                                                             4      0   100%
.tox/py37/lib/python3.7/site-packages/cachito/web/errors.py                                                          19      0   100%
.tox/py37/lib/python3.7/site-packages/cachito/web/manage.py                                                          20     20     0%
.tox/py37/lib/python3.7/site-packages/cachito/web/migrations/env.py                                                  32     10    69%
.tox/py37/lib/python3.7/site-packages/cachito/web/migrations/versions/15c4aa0c4144_rename_dependency_table.py        41     17    59%
.tox/py37/lib/python3.7/site-packages/cachito/web/migrations/versions/1714d8e3002b_map_deps_to_pkg.py                66     33    50%
.tox/py37/lib/python3.7/site-packages/cachito/web/migrations/versions/193baf9d7cbf_config_file_base64.py             20      7    65%
.tox/py37/lib/python3.7/site-packages/cachito/web/migrations/versions/3c208b05d703_request_state_fk.py               27     10    63%
.tox/py37/lib/python3.7/site-packages/cachito/web/migrations/versions/5854e700a35e_dependency_replacements.py        20      6    70%
.tox/py37/lib/python3.7/site-packages/cachito/web/migrations/versions/615c19a1cee1_add_npm.py                        23      7    70%
.tox/py37/lib/python3.7/site-packages/cachito/web/migrations/versions/71909d479045_add_submitted_by.py               14      3    79%
.tox/py37/lib/python3.7/site-packages/cachito/web/migrations/versions/9118b23629ef_add_indexes.py                    29     11    62%
.tox/py37/lib/python3.7/site-packages/cachito/web/migrations/versions/a655a299e967_request_flags.py                  12      2    83%
.tox/py37/lib/python3.7/site-packages/cachito/web/migrations/versions/b46cf36806d7_add_gomod_vendor_flag.py          16      3    81%
.tox/py37/lib/python3.7/site-packages/cachito/web/migrations/versions/c8b2a3a26191_initial_migration.py              22      6    73%
.tox/py37/lib/python3.7/site-packages/cachito/web/migrations/versions/cb6bdbe533cc_add_env_var_types.py              21      4    81%
.tox/py37/lib/python3.7/site-packages/cachito/web/migrations/versions/cdf17fad3edb_request_env_vars.py               12      2    83%
.tox/py37/lib/python3.7/site-packages/cachito/web/migrations/versions/fdd6d6978386_add_request_package_table.py      16      4    75%
.tox/py37/lib/python3.7/site-packages/cachito/web/models.py                                                         385     15    96%
.tox/py37/lib/python3.7/site-packages/cachito/web/utils.py                                                           12      0   100%
.tox/py37/lib/python3.7/site-packages/cachito/web/wsgi.py                                                             4      4     0%
.tox/py37/lib/python3.7/site-packages/cachito/workers/__init__.py                                                     0      0   100%
.tox/py37/lib/python3.7/site-packages/cachito/workers/cleanup_job.py                                                 50      8    84%
.tox/py37/lib/python3.7/site-packages/cachito/workers/config.py                                                     116      5    96%
.tox/py37/lib/python3.7/site-packages/cachito/workers/errors.py                                                       2      0   100%
.tox/py37/lib/python3.7/site-packages/cachito/workers/nexus.py                                                      149      2    99%
.tox/py37/lib/python3.7/site-packages/cachito/workers/paths.py                                                       22      0   100%
.tox/py37/lib/python3.7/site-packages/cachito/workers/pkg_managers/__init__.py                                        0      0   100%
.tox/py37/lib/python3.7/site-packages/cachito/workers/pkg_managers/general.py                                        69      6    91%
.tox/py37/lib/python3.7/site-packages/cachito/workers/pkg_managers/general_js.py                                    170      0   100%
.tox/py37/lib/python3.7/site-packages/cachito/workers/pkg_managers/gomod.py                                         191     16    92%
.tox/py37/lib/python3.7/site-packages/cachito/workers/pkg_managers/npm.py                                           135     10    93%
.tox/py37/lib/python3.7/site-packages/cachito/workers/requests.py                                                    21      2    90%
.tox/py37/lib/python3.7/site-packages/cachito/workers/scm.py                                                         73      0   100%
.tox/py37/lib/python3.7/site-packages/cachito/workers/tasks/__init__.py                                               3      0   100%
.tox/py37/lib/python3.7/site-packages/cachito/workers/tasks/celery.py                                                 7      0   100%
.tox/py37/lib/python3.7/site-packages/cachito/workers/tasks/general.py                                               59      3    95%
.tox/py37/lib/python3.7/site-packages/cachito/workers/tasks/gomod.py                                                 32      3    91%
.tox/py37/lib/python3.7/site-packages/cachito/workers/tasks/npm.py                                                   92      0   100%
-------------------------------------------------------------------------------------------------------------------------------------
TOTAL                                                                                                              2440    249    90%
```

Reply:
You want `--cov=cachito` so it ignores unit tests. 

Your latest comment actually shows code coverage being counted, just in the wrong directory. Set `usedevelop = true` under `[testenv]`. This will cause the files under cachito folder to be used.

Reply:
Yes, the idea was to check the files unted `.tox` and fetch the coverage from there. However, Lui's approach is cleaner and works as well!

Thanks, Lui!

Reply:
@lcarva @athos-ribeiro Thank you, guys!  Done :)